Solar-System.rb

```diff
@@ -0,0 +1,146 @@
+# Solar System project -create an object which contains a
+# collection of other objects
+
+require 'colorize'
+
+class SolarSystem
+  # create a constructor to add planet name
+  def initialize(planets)
+    @planets = planets
+  end
+  
+  attr_accessor :planets
+  
+  def list
+    num = 1
+    @planets.each do |arr|
+      print "\n#{num}. #{arr.summary} ".cyan
+      num += 1
+    end
+  end
+end
+
+# building multiple classes and composing these together
+class Planet
+  attr_accessor :name, :color, :orbital, :diameter, :distance
+  
+  # create a constructor to add planet attributes
+  def initialize(name, color, orbital, diameter, distance)
+    @name = name
+    @color = color
+    @orbital = orbital
+    @diameter = diameter
+    @distance = distance
+  end
+  # list attributes for each planet
+  def summary
+    " #{@name}\tColor: #{@color}, Orbital: #{@orbital}
+    \t\tDiameter(km): #{@diameter}, Distance(km): #{@distance}\n"
+  end
+  
+end
+
+# Wave 2/3 "hashes" with 5 attributes now variables
+mercury = Planet.new("Mercury", "silver", 0.24, 4879, 57909227)
+venus = Planet.new("Venus", "beige", 0.616, 12104, 108209475)
+earth = Planet.new("Earth", "green", 1, 12756, 149598262)
+mars = Planet.new("Mars", "red", 1.9, 6792, 227943824)
+jupiter = Planet.new("Jupiter","brown", 11.9, 142984,778340821)
+neptune = Planet.new("Neptune", "blue", 164.8, 49528, 4498396441)
+
+# Array for SolarSystem class
+planets = [ mercury, venus, mars, earth, jupiter, neptune ]
+
+solar = SolarSystem.new(planets)
+
+# show a list of the planets
+puts "\nHERE IS OUR LIST OF PLANETS:\n".cyan
+
+num = 1
+
+planets.each do |x|
+  puts "#{num}. #{x.name}".cyan
+  num += 1
```

Comment:
This is another place where it might make sense to define a method on `SolarSystem` rather than accessing the list of planets directly.